app/modules/core/module.go

```diff
@@ -1,9 +1,16 @@
 package core
 
-import "github.com/etcinit/phabulous/app/interfaces"
+import (
+	"github.com/etcinit/phabulous/app/interfaces"
+	"github.com/jacobstr/confer"
+	"github.com/Sirupsen/logrus"
+)
 
 // Module provides development/debugging commands.
-type Module struct{}
+type Module struct{
+	Config *confer.Config
+	Logger *logrus.Logger
```

Comment:
Where is this `Logger` used *if* `LookupCommand` does not have a logger?

Reply:
got rid of of the logger